### ISSUE TITLE
PHPUnit broke after updated to laravel 5.5.29

### ISSUE THREAD

**Issue by riseno** [apiato/apiato#365]
## Expected Behavior
Should be authenticated when running phpunit

## Actual Behavior
Fails when make call with the endpoint require authentication

## Steps to Reproduce the Problem

  1. composer update to laravel, make sure laravel framework version is 5.5.29
  2. run phpunit

## Specifications

  - Apiato Version: 7.2.*

## Additional Details

```sh
1) App\Containers\Merchant\UI\API\Tests\Functional\DeleteMerchantTest::testDeleteExistingMerchant
Illuminate\Auth\AuthenticationException: Unauthenticated.

/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Auth/Middleware/Authenticate.php:66
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Auth/Middleware/Authenticate.php:41
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:149
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
/Users/ryanchan/repository/valoot/vts/app/Ship/Middlewares/Http/ValidateJsonContent.php:41
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:149
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:102
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Router.php:660
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Router.php:635
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Router.php:601
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Router.php:590
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:176
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:30
/Users/ryanchan/repository/valoot/vts/vendor/barryvdh/laravel-debugbar/src/Middleware/InjectDebugbar.php:58
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:149
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
/Users/ryanchan/repository/valoot/vts/vendor/barryvdh/laravel-cors/src/HandleCors.php:37
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:149
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
/Users/ryanchan/repository/valoot/vts/vendor/fideloper/proxy/src/TrustProxies.php:56
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:149
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:30
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:149
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:30
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:149
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php:27
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:149
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php:46
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:149
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
/Users/ryanchan/repository/valoot/vts/app/Containers/Debugger/Middlewares/RequestsMonitorMiddleware.php:27
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:149
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:102
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:151
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:116
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:345
/Users/ryanchan/repository/valoot/vts/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:317
/Users/ryanchan/repository/valoot/vts/vendor/apiato/core/Traits/TestsTraits/PhpUnit/TestsRequestHelperTrait.php:104
/Users/ryanchan/repository/valoot/vts/app/Containers/Merchant/UI/API/Tests/Functional/DeleteMerchantTest.php:29
```

**Comment by Kyslik** [comment]
maybe related to https://github.com/laravel/framework/issues/22808

**Comment by riseno** [comment]
ahhhh....   that's sad. 

thanks for that information.

**Comment by riseno** [comment]
closing this issue due to the problem is fixed laravel/framework#22815
